tools/stress-cluster/services/chaos-watcher/CustomResourceDefinition.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using k8s;
+using k8s.Models;
+
+namespace chaos_watcher
```

Comment:
This is not very .NET like. For .NET this should be "Chaos.Watcher"

Reply:
yeah I was thinking I had this wrong :) I think I'm going to change it to `Stress.Watcher` then. Does the directory naming need to match the namespace, conventionally?

Reply:
Yes usually the directory structure beyond the root does match the namespaces and the file names match the class name in them.